addons/stock_account/models/stock_lot.py

```diff
@@ -0,0 +1,136 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from ast import literal_eval
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_compare, float_round
+
+
+class StockLot(models.Model):
+    _inherit = 'stock.lot'
+
+    value_svl = fields.Float(compute='_compute_value_svl', compute_sudo=True)
+    quantity_svl = fields.Float(compute='_compute_value_svl', compute_sudo=True)
+    avg_cost = fields.Monetary(string="Average Cost", compute='_compute_value_svl', compute_sudo=True, currency_field='company_currency_id')
+    total_value = fields.Monetary(string="Total Value", compute='_compute_value_svl', compute_sudo=True, currency_field='company_currency_id')
+    company_currency_id = fields.Many2one('res.currency', 'Valuation Currency', compute='_compute_value_svl', compute_sudo=True)
+    stock_valuation_layer_ids = fields.One2many('stock.valuation.layer', 'lot_id')
+    standard_price = fields.Float(
+        "Cost", company_dependent=True,
+        digits='Product Price', groups="base.group_user",
+        help="""Value of the lot (automatically computed in AVCO).
+        Used to value the product when the purchase cost is not known (e.g. inventory adjustment).
+        Used to compute margins on sale orders."""
+    )
+
+    @api.depends('stock_valuation_layer_ids', 'product_id.lot_valuated')
+    @api.depends_context('to_date', 'company')
+    def _compute_value_svl(self):
```

Comment:
![image](https://github.com/odoo/odoo/assets/5512395/0df81814-5a89-4159-86dc-b70ee1324aa2)
It didn't trigger on update standard_price

Reply:
![image](https://github.com/odoo/odoo/assets/5512395/24289788-fd80-480f-b1cc-4772cb9950e1)
It didn't trigger at all